src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,98 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.24.0.dev3 (12/13/2019)
+------------------------
+
+New Features
+~~~~~~~~~~~~
+
+* A rule to create a pex from a target closure. (#8811)
```

Comment:
This isn’t user facing, beyond rule authors, which I think we are acting like is not public API given how quickly we are to break things. 

I’d reclassify to internal changes, although acknowledge an argument to keep it here

Reply:
Done.